doc/mz_os.md

```
@@ -463,6 +465,27 @@ if (test_utf8) {
 }
 ```
 
+### mz_os_utf8_string_is_valid
+
+Checks if a string contains a valid UTF-8 byte sequence.
+
+**Arguments**
+|Type|Name|Description|
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)